mission_control/navigator_missions/nav_missions/scan_the_code.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+import txros
+import numpy as np
+from navigator_msgs.srv import ObjectDBSingleQuery, ObjectDBSingleQueryRequest
+
+
+@txros.util.cancellableInlineCallbacks
+def main(navigator):
+    navigator.change_wrench("autonomous")
+    serv = navigator.nh.get_service_client("/database/single", ObjectDBSingleQuery)
```

Comment:
Oooh good idea! I will do this in V2, I am making a todo list for it currently. You can see it in my TODO file.